### DISCUSSION TITLE
alternative payoutScheme than PPLNS? and what does payoutSchemeConfig factor effect?

### DISCUSSION THREAD

**Discussion by mboyd1** [coinfoundry/miningcore#1356] [Q&A] [answered]
bitcoin-family coin, I have this in config:

                "payoutScheme": "PPLNS",
                "payoutSchemeConfig": {
                    "factor": 0.5

I understand what PPLNS is but are there other options built in?  and what other payoutSchemeConfig options are there? and what does this 'factor' affect

**Comment by mboyd1** [comment]
I see in previous discussions that 'PROP' payout scheme is perhaps on the roadmap, any other schemes in the works?

also saw that 'factor' has something to do with payouts being strung across multiple blocks when there are few shares submitted towards a solved block, and that 0.5 is probably fine for a small pool and 2 for a big pool

**Comment by seperatis** [comment, marked as answer]
I think Oliver's previous explanation of this will help you understand factor size.
Please see: https://github.com/oliverw/miningcore/issues/52

Payment Schemes are here: https://github.com/oliverw/miningcore/tree/master/src/Miningcore/Payments/PaymentSchemes
